README.md

```diff
@@ -1,5 +1,8 @@
 # project-sandbox
-a playground for experimentation
+* Project Sandbox will explore mapping the uses of publicly-owned vacant properties owned by the City of Los Angeles.
+* Project Sandbox is an example of using github resources to develop, document, track and manage and deploy civic projects with involved participation and collaboration from the general public from the beginning.
+* MaptimeLA and the city will hold a series of workshops to teach and bring the project together.
```

Comment:
MaptimeLA and the city will hold a series of workshops to teach the public how to use government data to create maps and answer questions.